### PR TITLE
Support for AcceptMergeRequest parameters

### PR DESCRIPTION
**Pull request by jereksel**
Implements #81

### CODE REVIEW

File: merge_requests.go [hunk 1]
```diff
@@ -286,12 +299,21 @@ func (s *MergeRequestsService) UpdateMergeRequest(
...
-	u := fmt.Sprintf("projects/%s/merge_request/%d/merge", url.QueryEscape(project), mergeRequest)
+
+	v, err := query.Values(opt)
+
+	if err != nil {
+		return nil, nil, err
+	}
+
+	//For some reason putting struct to body doesn't work
+	u := fmt.Sprintf("projects/%s/merge_request/%d/merge?%s", url.QueryEscape(project), mergeRequest, v.Encode())
```
Comment (svanharmelen) [inline merge_requests.go]: Could you explain what is not working? As this is done for all other calls successfully, I wonder if it's a bug in the Gitlab API? And if so, maybe it's already fixed in the latest version(s)?

**Comment by svanharmelen** [comment]
@jereksel thanks for the PR! Just have a question about the error you get, the code itself looks fine...

File: merge_requests.go (hunk 1 above)
Comment (jereksel) [inline merge_requests.go]: I'm testing in on 8.11.6. Also I don't userstand why putting it in body would work in the first place - Gitlab docs say that parameters should be put into URL.

**Comment by svanharmelen** [comment]
@jereksel I tested this in a few different ways just now, and I do not get any error when using the logic using in the rest of this package (so putting the options in the body instead of the URL). So I created a new PR (#104) in which I cherry picked your commit and updated the code to make in inline with the rest.

Thanks for your PR and sorry for the long delay in looking into this one!!

**Pull request closed without merging**